### DISCUSSION TITLE
Running any other homebrew programs with webMAN-MOD Enabled on HEN PS3 Results in black screen

### DISCUSSION THREAD

**Discussion by qfoxb** [aldostools/webMAN-MOD#758] [Q&A] [answered]
I'm trying to run mmCM on my HEN PS3 (4.89.1 HFW) and whenever i run mmCM right after enabling HEN 3.1.1 and webman loads, the whole console freezes and doesn't respond to any input and i have to force shutdown the system. Is there any fix? Edit: The HDD Light also goes dead.

**Comment by aldostools** [comment, marked as answer]
Set the fan control to manual speed or SYSCON.
Then save the settings in /setup.ps3

**Reply by qfoxb** [reply]
I actually had it on manual speed, but what ended up working was just Re-installing HFW, repair filesystem then rebuild database. (Plus, reinstalling webMAN MOD as well) so i'll just mark it as answered

**Reply by jimhew** [reply]
I just run mine on SYSCON and monitor the temps. I also deleted multiMan as that was affecting it too. 

Mine only freezes now either when I bring up the temperatures (start+select) when searching in WMAN games folders or when I use a USB controller / my TV remote to search WMAN games.
